library/src/scala/tasty/reflect/TreeUtils.scala

```diff
@@ -97,13 +99,25 @@ trait TreeUtils
       case TypeTree.Applied(tpt, args) => foldTypeTrees(foldTypeTree(x, tpt), args)
       case TypeTree.ByName(result) => foldTypeTree(x, result)
       case TypeTree.Annotated(arg, annot) => foldTree(foldTypeTree(x, arg), annot)
+      case TypeTree.TypeLambdaTree(typedefs, arg) => foldTrees(foldTypeTree(x, arg), typedefs)
```

Comment:
It should be `foldTypeTree(foldTrees(x, typedefs), arg)`

Reply:
Fixed